slime-modules/lazyload/controllers/servicefence_controller.go

```diff
@@ -329,7 +329,11 @@ func newSidecar(vhost *lazyloadv1alpha1.ServiceFence, env *bootstrap.Environment
 	// 需要加入一条根namespace的策略
 	host = append(host, env.Config.Global.IstioNamespace+"/*")
 	host = append(host, env.Config.Global.SlimeNamespace+"/*")
-	host = append(host, fmt.Sprintf("*/global-sidecar.%s.svc.cluster.local", vhost.Namespace))
+	// check whether global-sidecar is deployed in the same namespace
+	// if so, init config of sidecar will adds */global-sidecar.${svf.ns}.svc.cluster.local
+	if _, err := env.K8SClient.AppsV1().Deployments(vhost.Namespace).Get("global-sidecar", metav1.GetOptions{}); err == nil {
```

Comment:
get this info from config rather than checking it at run time

Reply:
add "global-sidecar-mode" in global.misc